map/templates/map/add_spot.html

```diff
@@ -61,6 +66,10 @@ <h3 class="text-center">New Post</h3>
             </option>
           </select>
         </div>
+        <div class="form-group py-1">
```

Comment:
Shows up even if the owner is not verified

Should this be conditionally rendered only if the owner is verified for Private Parking?

Reply:
Also, can we add validation to the Capacity Input field so it must be >= 1?